File: Concurs Inot C#/Persoana.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public class Persoana : Entity<int>
    {
        public string Nume { get; set; }
        public int Varsta { get; set; }
        public Persoana(int id,string nume, int varsta)
        {
            base.Id = id;
            Nume = nume;
            Varsta = varsta;
        }

        public override string ToString()
        {
            return Nume + " " + Varsta + " ani";
        }
    }
}

File: Persoana.java
package me.model;

import java.io.Serializable;

abstract public class Persoana extends Entity<Integer> implements Serializable{
    private String nume;
    private int varsta;

    public Persoana(Integer id, String nume, int varsta) {
        super.setId(id);
        this.nume = nume;
        this.varsta = varsta;
    }

    public String getNume() { return nume; }

    public void setNume(String nume) { this.nume = nume; }

    public int getVarsta() { return varsta; }

    public void setVarsta(int varsta) { this.varsta = varsta; }

    @Override
    public String toString() {
        return nume+" "+varsta+" ani ";
    }
}
